starter-code/basic-algorithms.js

```diff
@@ -7,6 +7,7 @@ console.log ("The navigator's name is", hacker2);
 
```

Comment:
les falto agregar esta primer console     `console.log("The driver's name is ",hacker1) `